### ISSUE TITLE
Bumblebee deprecation?

### ISSUE THREAD

**Issue by hadronized** [Bumblebee-Project/Bumblebee#607]
This is not a regular issue; more a general question.

Since I haven’t been able to get bumblebee work on my laptop (see #600 #605) I’ve been wandering around and ended up on [this](https://wiki.archlinux.org/index.php/PRIME). I’m now using **nouveau** (since **nvidia** driver is not supported by the kernel prime yet – I reckon it won’t ever be), and everything is pretty usuable (I still have troubles with powermanagement through the `/sys/class/drm/card0/device/pstate`, but that’s a detail I’m sure it’ll be solved sooner or later).

What’s the purpose of bumblebee then if **prime** gets better and better? Do you guys work with the prime guys? I’d love to contribute to any optimus related project – I’m a C/C++/Haskell programmer. I guess it requires specific and hard/tough skills, but I’m motivated.

**Comment by ArchangeGabriel** [comment]
Indeed, after looking at it again, PRIME also achieve power saving.

But Bumblebee is still needed, for those that have a computer where nouveau doesn’t work currently (me), or that need latest OpenGL (currently 4.0+, since Mesa stops at 3.3), so all people that rely in a way or another to the NVIDIA proprietary driver (and I suppose CUDA is another use case of this, I’m not aware of the fact it can work with nouveau). But in the end, the goal is to get ride of Bumblebee everywhere, sure.
